products/urls.py

```diff
@@ -1,6 +1,6 @@
-from django.urls import path
-
+from django.urls    import path
+from products.views import HostEnrollment
 
 urlpatterns = [
-    
+    path('', HostEnrollment.as_view()),
```

Comment:
`POST /host` http method와 URI만으로 해당 데이터에 대한 엑션을 파악할 수 있기 때문에 Enrollment라는 용어는 삭제해도 될 것 같습니다. 그리고 나중에 host 정보를 받아오는 get()를 구현해야될 때, HostEnrollment Class 안에 get method로 구현해야 되기 때문에 아래 class명도 `Class Host`로 수정해주세요~

```suggestion
    path('', Host.as_view()),
```

Reply:
네 수정하겠습니다~